components/discount/registerDiscount.tsx

```diff
@@ -59,6 +72,10 @@ const RegisterDiscount: FunctionComponent<RegisterDiscountProps> = ({
   priceInEth,
   mailGroups,
   goBack,
+  couponCode,
+  couponHelper,
+  banner = "/visuals/register.webp",
+  getCustomCalls,
```

Comment:
What do you think of creating a `freeRegister.tsx` component based on this one instead of reusing it ? 

you wouldn't need to pass the calls in props ... 

I think it would be clearer tbh